harmonica/tests/test_prism.py

```diff
@@ -52,6 +52,23 @@ def test_invalid_prisms():
         _check_prisms(np.atleast_2d([w, e, s, n, top, bottom]))
 
 
+@pytest.mark.use_numba
+def test_disable_checks():
+    "Check if the disable_checks flag works properly"
+    valid_prism = [0, 10, 0, 10, -10, 0]
+    invalid_prism = [0, 10, 0, 10, 0, -10]
+    density = 100
+    coordinates = [0, 0, 0]
+    # Check if an invalid prism doesn't raise an error with the disable_checks flag set to True
+    invalid_result = prism_gravity(
+        coordinates, invalid_prism, density, field="potential", disable_checks=True
+    )
+    # Check if the invalid prism generates a potential field inverse
+    # than the one generated by the valid prism
+    valid_result = prism_gravity(coordinates, valid_prism, density, field="potential")
+    assert invalid_result == -valid_result
```

Comment:
```suggestion
    npt.assert_allclose(invalid_result, -valid_result)
```

When comparing floating point numbers, it's always better to use `numpy.testing.assert_allclose` or `numpy.testing.assert_almost_equal` because they let you set a desired precision. Comparing with `==` tends to fail on different OSs or when packages update (I've suffered from this too many times in the past :-)